lib/core/src/sdk.rs

```diff
@@ -301,28 +305,40 @@ impl LiquidSdk {
     async fn check_chain_swap_expiration(&self, chain_swap: &ChainSwap) -> Result<()> {
         if chain_swap.user_lockup_tx_id.is_some() && chain_swap.refund_tx_id.is_none() {
             match chain_swap.direction {
+                Direction::Incoming => {
+                    let swap_script = chain_swap.get_lockup_swap_script()?.as_bitcoin_script()?;
+                    let current_height =
+                        self.bitcoin_chain_service.lock().await.tip()?.height as u32;
+                    let locktime_from_height =
+                        LockTime::from_height(current_height).map_err(|e| {
+                            PaymentError::Generic {
+                                err: format!(
+                                    "Error getting locktime from height {current_height:?}: {e}",
+                                ),
+                            }
+                        })?;
+
+                    info!("Checking Chain Swap {} expiration: locktime_from_height = {locktime_from_height:?},  swap_script.locktime = {:?}", chain_swap.id, swap_script.locktime);
+                    if swap_script.locktime.is_implied_by(locktime_from_height) {
```

Comment:
What are we checking here? Not sure is is correct as we need to check that the current tip minus the height of the lockup tx confirmation block is less than the timeout in blocks.

Reply:
This is doing essentially the same as `utils::is_locktime_expired` but for bitcoin.

```
   pub fn is_implied_by(&self, other: LockTime) -> bool {
        use LockTime::*;

        match (*self, other) {
            (Blocks(this), Blocks(other)) => this.value() <= other.value(),
            (Time(this), Time(other)) => this.value() <= other.value(),
            _ => false, // Not the same units.
        }
    }
```

Reply:
Then maybe we can replace `utils::is_locktime_expired` with `is_implied_by` in the other places, too?

Reply:
Seems the elements LockTime doesn't have this function

Reply:
I see, the lock height is absolute, I missed that.